junit-platform-engine/src/main/java/org/junit/platform/engine/support/hierarchical/ThrowableCollector.java

```diff
@@ -8,22 +8,27 @@
  * http://www.eclipse.org/legal/epl-v20.html
  */
 
-package org.junit.jupiter.engine.execution;
+package org.junit.platform.engine.support.hierarchical;
 
-import static org.apiguardian.api.API.Status.INTERNAL;
+import static org.apiguardian.api.API.Status.MAINTAINED;
+import static org.junit.platform.commons.util.BlacklistedExceptions.rethrowIfBlacklisted;
+import static org.junit.platform.engine.TestExecutionResult.aborted;
+import static org.junit.platform.engine.TestExecutionResult.failed;
+import static org.junit.platform.engine.TestExecutionResult.successful;
 
 import org.apiguardian.api.API;
-import org.junit.jupiter.api.function.Executable;
 import org.junit.platform.commons.util.ExceptionUtils;
 import org.junit.platform.commons.util.Preconditions;
+import org.junit.platform.engine.TestExecutionResult;
+import org.opentest4j.TestAbortedException;
```

Comment:
This has the consequence of introducing a mandatory dependency on the OTA for any implementation of `HierarchicalTestExecutor` even if such a `TestEngine` does not use or even support exceptions from the OTA.

Thus, I'm not so sure we would want to do that.

Perhaps an alternative would be to allow a caller of the relocated `ThrowableCollector` to specify what the caller considers to be an "overridable exception".